newsletter/README.md

```diff
@@ -3,12 +3,12 @@
 | --- | --- | --- |
 |[OSPO Newsletter 1](https://ospo-news.ghost.io/ospo-news-from-the-todo-group/)|[OSPO Newsletter 11](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-11/)|[OSPO Newsletter 21](https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-21/)|
 |[OSPO Newsletter 2](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-2/)|[OSPO Newsletter 12](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-12/)|[OSPO Newsletter 22](https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-22/)|
-|[OSPO Newsletter 3](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-3/)|[OSPO Newsletter 13](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-13/)|
-|[OSPO Newsletter 4](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-4/)|[OSPO Newsletter 14](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-14/)|
-|[OSPO Newsletter 5](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-5/)|[OSPO Newsletter 15](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-15/)|
-|[OSPO Newsletter 6](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-6/)|[OSPO Newsletter 16](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-16/)|
-|[OSPO Newsletter 7](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-7/)|[OSPO Newsletter 17](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-17/)|
-|[OSPO Newsletter 8](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-8/)|[OSPO Newsletter 18](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-18/)|
+|[OSPO Newsletter 3](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-3/)|[OSPO Newsletter 13](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-13/)|[OSPO Newsletter 23](https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-23/)|
+|[OSPO Newsletter 4](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-4/)|[OSPO Newsletter 14](https://ospo-news.ghost.io/ospo-news-from-the-todo-group-issue-14/)|[OSPO Newsletter 24](https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-24/)|
```

Comment:
I got an issue with the issue 24. The others are working fine.

![Screenshot 2024-01-12 at 15 44 55](https://github.com/todogroup/ospology/assets/5110813/efcfd597-9137-4654-999a-b0300a6852fe)

Reply:
Nice catch.  The correct link for Issue 24 is [https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-23-2/](https://ospo-news.ghost.io/osponews-from-the-todo-group-issue-23-2/).

Reply:
Thank you, merging the PR.